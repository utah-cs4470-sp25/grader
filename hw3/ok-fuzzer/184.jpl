print "bR"
write image false to ""
let n = false
print ""
let S = c7d_
read image "" to Z
time show [[ u9T, 8 , 8 ,I]]
print "mA"
let c = .9
