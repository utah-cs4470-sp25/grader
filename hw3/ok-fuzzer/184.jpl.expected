(PrintCmd "bR")
(WriteCmd (FalseExpr) "")
(LetCmd (VarLValue n) (FalseExpr))
(PrintCmd "")
(LetCmd (VarLValue S) (VarExpr c7d_))
(ReadCmd "" (VarLValue Z))
(TimeCmd (ShowCmd (ArrayLiteralExpr (ArrayLiteralExpr (VarExpr u9T) (IntExpr 8) (IntExpr 8) (VarExpr I)))))
(PrintCmd "mA")
(LetCmd (VarLValue c) (FloatExpr 0))
